src/CommonAssemblyInfo.cs

```diff
@@ -3,9 +3,9 @@
 using System.Resources;
 using System.Runtime.InteropServices;
 
-[assembly: AssemblyVersion("2.1.0")]
-[assembly: AssemblyFileVersion("2.1.0")]
-[assembly: AssemblyInformationalVersion("2.1.0")]
+[assembly: AssemblyVersion("2.0.1")]
+[assembly: AssemblyFileVersion("2.0.1")]
+[assembly: AssemblyInformationalVersion("2.0.1")]

```

Comment:
#733 is the "release checklist issue", not a release. I was looking for something like https://github.com/FakeItEasy/FakeItEasy/releases/tag/untagged-501bff0d4409887cd5ef, only more "2.0.1"-like.

> TBH I don't think I've ever used `set_version[new_version]`

Well, you've not actually initiated many releases since, well, since I showed up. Certainly not since the rake target existed.

Does this also mean that you didn't use `rake create_milestone` to create the milestone and release issue checklist? I'd feel better if I knew you didn't, and the lack of GitHub Release was because you didn't make one, not because the rake target failed without alerting you.